ideas/122-sob-increase_platform_usage.md

```diff
@@ -0,0 +1,60 @@
+## Preamble
+
+    Idea: 122-sob-increase_platform_usage
+    Title: Open Bounty - Increase Platform Usage
+    Status: Draft
+    Created: 2018-04-04
+    Related: 
+
+## Summary
+
+To identify the successful use of a platform, it's important and imperative to be able to measure its usage. As the Open Bounty platform continues to grow and evolve, its important to be able to track its growth, and make data-driven decisions.
+
+One of the key objectives of Open Bounty is to increase the usage of the platform. The purpose of this swarm is to build the tools and means necessary to capture these metrics and be able to report on them on a regular basis.
+
+## Swarm Participants
+
+- Lead Contributor\PM: @arash009 (Arash)
+- Contributor: @siphiuel (Vitaliy)
+- Testing and QA: 
+
+## Product Overview
+
+This swarm will create the mechanism to capture **three** specific metrics on the usage of Open Bounty (specific criteria for each metric yet to be defined):
+
+- **Value transacted** — This metric is to capture the **$USD equivalent** of total tokens that have been paid out in a given timeframe (daily, weekly, monthly, annual)
+- **Active contributors** — This metric is to capture the **Monthly Active Contributors** against each organization.
+- **Unique visitors** — This metric is to capture the **Unique Monthly Visitors** to openbounty.status.im
+
+The collection of these metrics is to be automated, however their reporting and sharing can be manual, or basic automation, with future iterations looking at full automation/analytics.
+
+## Success Criteria
+
+#### Milestone 1
```

Comment:
Adding some dates/timeframes to these milestones might be good just to give a rough idea. Can be changed/updated in swarm updates as appropriate.